content/docs/reference/crosswalk/aws/ecs.md

```diff
@@ -57,20 +57,21 @@ const nginx = new awsx.ecs.FargateService("nginx", {
 export const url = lb.endpoint.hostname;
 ```
 
-After deploying this program, we can access our 2 Nginx web servers behind our load balancer:
+After deploying this program, we can access our two  NGINX web servers behind our load balancer:
 
 ```bash
-$ curl https://$(pulumi stack output url)
+$ curl http://$(pulumi stack output)
```

Comment:
`http` seems correct to change here, but I believe the `... url` bit is still needed.